twistlock/tests/test_twistlock.py

```diff
@@ -0,0 +1,70 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import mock
+import json
+
+from datadog_checks.twistlock import TwistlockCheck
+
+customtag = "custom:tag"
+
+instance = {
+    'url': 'http://localhost:8081',
+    'tags': [customtag],
+    'ssl_verify': False
+}
+
+METRICS = [
+    'twistlock.registry.cve.details',
+    'twistlock.registry.cve.count',
+    'twistlock.registry.compliance.count',
+    'twistlock.registry.size',
+    'twistlock.registry.layer_count',
+    'twistlock.images.cve.details',
+    'twistlock.images.cve.count',
+    'twistlock.images.compliance.count',
+    'twistlock.images.size',
+    'twistlock.images.layer_count',
+    'twistlock.hosts.cve.details',
+    'twistlock.hosts.cve.count',
+    'twistlock.hosts.compliance.count',
+    'twistlock.containers.compliance.count',
+]
+
+HERE = os.path.dirname(os.path.abspath(__file__))
```

Comment:
let's use `get_here` from utils.

Reply:
Where is that? I can't find it when I search

Reply:
I found it, had to rebase